vapix-sh/app/manifest.json

```diff
@@ -0,0 +1,17 @@
+{
+    "schemaVersion": "1.7.2",
+    "acapPackageConf": {
+        "setup": {
+            "appName": "vapix_example_sh",
```

Comment:
Is there a reason why we don't set `"architecture": "all"`?

Reply:
Good point. We should have that instead.

Reply:
Happy to hear we understand the architecture option the same way